crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -130,6 +130,11 @@ def adminrepocheck
 
         {}.tap do |ret|
           if zypper_stream["message"] =~ /^System management is locked/
+            upgrade_status.end_step(
+              false,
+              data: zypper_stream["message"],
+              help: "Make sure zypper is not running and try again."
+            )
```

Comment:
`data` and `help` must actually be inside another hash, see https://github.com/crowbar/crowbar-core/pull/985.